app/components/Sidebar/Sidebar.tsx

```diff
@@ -1,31 +1,73 @@
-import { List, useMantineTheme } from "@pokt-foundation/pocket-blocks"
+import { List, MediaQuery, useMantineTheme } from "@pokt-foundation/pocket-blocks"
+import { useState } from "react"
+import SidebarRightButton from "../Icons/SidebarRightButton"
 import LinksGroup, { LinksGroupProps } from "../LinksGroup/LinksGroup"
+import SidebarLeftButton from "../Icons/SidebarLeftButton"
 
 type SidebarProps = {
   data: LinksGroupProps[]
 }
 
 export function Sidebar({ data }: SidebarProps) {
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false)
   const theme = useMantineTheme()
 
   return (
-    <List
-      unstyled
-      withPadding
-      m={0}
-      sx={{
-        height: "100vh",
-        padding: "0 8px 8px 0",
-        width: 300,
-        top: 0,
-        position: "sticky",
-        background: theme.colors.navy[7],
-        overflowY: "auto",
-      }}
-    >
-      {data.map((item) => (
-        <LinksGroup {...item} key={item.label} />
-      ))}
-    </List>
+    <>
+      <MediaQuery smallerThan="md" styles={{ display: "none" }}>
+        <List
+          unstyled
+          withPadding
+          m={0}
+          sx={{
+            height: "100vh",
+            padding: "0 8px 8px 0",
+            width: 300,
+            top: 0,
+            position: "sticky",
+            background: theme.colors.navy[7],
+            overflowY: "auto",
+          }}
+        >
+          {data.map((item) => (
+            <LinksGroup {...item} key={item.label} />
+          ))}
+        </List>
+      </MediaQuery>
+      <MediaQuery largerThan="md" styles={{ display: "none" }}>
+        <List
```

Comment:
I remember the discussion to use https://mantine.dev/core/drawer
Did you prefer to use this because of some limitations or were there any drawbacks with using the Mantine's Drawer component?

Reply:
I honestly forgot about it. I tried to do something similar to what we already have on the Header